plutarch.cabal

```diff
@@ -63,88 +59,93 @@ common c
     ScopedTypeVariables
     StandaloneDeriving
     StandaloneKindSignatures
-    NoStarIsType
     TraditionalRecordSyntax
     TupleSections
     TypeApplications
+    TypeFamilies
     TypeOperators
     TypeSynonymInstances
+    ViewPatterns
+
+  -- poor man's GHC2021
```

Comment:
nit: This comment is out of context now, probably could be removed.

Reply:
:ok_hand: